### DISCUSSION TITLE
Use 2 hdd for frigate recording

### DISCUSSION THREAD

**Discussion by mdv0** [blakeblackshear/frigate#13471] [General]
I'm running Frigate on Docker and currently have two separate HDDs on my system. I don't want to combine them into a single volume in Linux, but I'd like to use both HDDs for storing Frigate media.

Ideally, I'd like to:

Use both HDDs equally for storing media files, or
Use the second HDD only when the first one is full.
I've mounted both drives at /media/hdd1 and /media/hdd2, and I’ve created directories for media storage on each drive.

Is there a way to configure the docker-compose.yml file (or any other configuration) to achieve this setup? Any suggestions on how to approach this?

Thanks in advance for your help!

**Comment by NickM-27** [comment]
you can use something like fuse.fs
